BrushUpCode/src/Coffee.java

```diff
@@ -0,0 +1,340 @@
+import java.util.Scanner;
+
+public class Coffee {
```

Comment:
@maria-salmasian  public mean available from everywhere, but your classes are in the same package, you can restrict accessibility a little bit

Reply:
@maria-salmasian this still exists